snap/snapcraft.yaml

```diff
@@ -13,6 +13,16 @@ base: core22
 architectures:
   - build-on: arm64
 
+slots:
+  gpiochip-slot:
+    interface: custom-device
+    custom-device: gpiochip
+    devices:
+      - /dev/gpiochip[0-4]
+    files:
+      read:
+        - /sys/devices/platform/soc/*.gpio/gpiochip[0-4]/dev
```

Comment:
Nice work. 
How about `gpio-dev` for the slot name and `gpio` for the plug?

Also, this is a breaking change on the connections. So make sure to update the docs and bump a major.

Reply:
Done! Major and documentation updated.